apps/frontend/src/components/Decorators/DynamicComponent.tsx

```diff
@@ -8,11 +8,7 @@ interface DynamicComponentProps {
 const DynamicComponent: React.FunctionComponent<DynamicComponentProps> = (props) => {
     const Component = (allComponent as Record<string, any>)[props.componentName];
 
-    return (
-        <div>
```

Comment:
I made this cause of the flex beahavior
I hope you take it in your mind (avoir ça en tête)

Reply:
Yes, the flex display behaviour is okay